src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -91,13 +105,19 @@ private protected void DisconnectHandler(NativeView? nativeView)
 			VirtualView = null;
 		}
 
+		static partial void UpdatingWidth(IViewHandler handler, IView view);
+
 		public static void MapWidth(IViewHandler handler, IView view)
 		{
+			UpdatingWidth(handler, view);
 			((NativeView?)handler.NativeView)?.UpdateWidth(view);
 		}
 
+		static partial void UpdatingHeight(IViewHandler handler, IView view);
+
 		public static void MapHeight(IViewHandler handler, IView view)
 		{
+			UpdatingHeight(handler, view);
 			((NativeView?)handler.NativeView)?.UpdateHeight(view);
 		}
```

Comment:
Is this the right value to be watching? Should we be tracking the actual Frame bounds set by the layout engine?